SAP/spartacus
# chore: Automatically bump our peerDependencies

QA steps:
- run release-it scripts for storefrontlib/cds (only libraries having peerDependencies in our libraries)
- set new version
- check if all our peerDependencies are updated in those package.json files
- check if those changes are commited by .release-it (check after commit step git log and the diff)
- abort release process
- revert bump commits locally

I used my fork of the @release-it/bumper, because the original wasn't working correctly if you wanted to change version in 2 places in the same file.

After merging this PR to both maintenance and develop branch we could remove the peerDependencies manual bump step.

Closes #7186
